x-pack/plugins/canvas/server/collectors/collector.ts

```diff
@@ -24,10 +23,13 @@ const collectors: TelemetryCollector[] = [workpadCollector, customElementCollect
   A usage collector function returns an object derived from current data in the ES Cluster.
 */
 export function registerCanvasUsageCollector(
-  usageCollection: UsageCollectionSetup,
-  core: CoreSetup
+  usageCollection: UsageCollectionSetup | undefined,
+  kibanaIndex: string
 ) {
-  const kibanaIndex = core.getServerConfig().get<string>('kibana.index');
+  if (!usageCollection) {
```

Comment:
Do you think this check should move to the plugin defintion instead of here? So we don't try to register unless we have the usage_collection plugin?